playbooks/common/openshift-cluster/upgrades/upgrade_scale_group.yml

```diff
@@ -47,7 +47,7 @@
     command: >
       {{ hostvars[groups.oo_first_master.0]['first_master_client_binary'] }} adm drain {{ l_kubelet_node_name | lower }}
       --config={{ openshift.common.config_base }}/master/admin.kubeconfig
-      --force --delete-local-data --ignore-daemonsets
+      --force --grace-period=30 --delete-local-data --ignore-daemonsets
```

Comment:
Default grace period is 30s afaik. It would be be configurable via an ansible var

Reply:
@vrutkovs I was not aware of that default grace period is 30s, thanks for the heads up. I created that PR cause my containers was not terminating at all, so i had to terminate them forcefully by hand at the playbook runtime. But i guess that is a special case for me. 

Thanks anyway.